### ISSUE TITLE
Doc: isArray : Wrong example

### ISSUE THREAD

**Issue by Gillespie59** [Gillespie59/eslint-plugin-angular#291]
the following example should fail I think

```javascript
angular.isArray(someArray);
```

**Comment by tilmanpotthof** [comment]
Why should it fail?

**Comment by Gillespie59** [comment]
According to the unit test : https://github.com/Gillespie59/eslint-plugin-angular/blob/master/test/typecheck-array.js

For testing array object,, this rule recommend the use of `angular.isArray()`

**Comment by tilmanpotthof** [comment]
The test `Array.isArray(someArray)` should fail, but `angular.isArray(someArray)` should not fail.

The example for `angular.isArray(someArray)` says: The following patterns are **not** considered problems;

... and that is correct, as far I a understand the rule.

**Comment by Gillespie59** [comment]
Oops sorry . You are right. 

We do not use `Array.isArray` in the demo page ?

**Comment by tilmanpotthof** [comment]
Yes, but I will add it as failing example.

**Comment by tilmanpotthof** [comment]
PR https://github.com/Gillespie59/eslint-plugin-angular/pull/300
